script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -34,15 +34,15 @@ If there were people in that building who hadn't yet evacuated, every single one
 <Page10>
 My body trembles at the unbelievable scene in front of me.
 I'm terrified at the prospect of what might happen next.
-It's just a coincidence that I'm still alive.//ruby dots <た|・><ま|・><た|・><ま|・>
+It's only by chance that I'm still alive.//ruby dots <た|・><ま|・><た|・><ま|・>
 If I had been behind Arcueid, I would've been frozen where I stood.
 <Page11>
 The expanding river of flames was a slow-moving disaster.
-People were running away faster than the fire was able to spread.
+People could run away faster than the fire was able to spread.
```

Comment:
> 広がっていく火の手より人間の逃げる足の方が早かった。

I think 'were running' is more accurate. Also, are we removing the 火の手 or should that be included?

Reply:
I guess I internal-tl'd `人間の逃げる足の方が早かった` as `the (fleeing legs) of humans were faster`, and naturalized it to could/were able to run away faster in context of the next few lines, since he's comparing the fact that people _could_ run away from the fire to the fact that people _cannot_ run from the ice. 

Re: 火の手, I think this is a different formulation than the 'hand', if that's what you were thinking of - 
> 火の手 (ひのて) (exp,n) (1) flame; blaze; fire; (2) (id) attack; aggression; 

or do you think we should just make the aggression clearer?

Reply:
I get what you mean and agree now. I would prefer 'people were able to run away' though.